lib/matplotlib/texmanager.py

```diff
@@ -103,33 +110,48 @@ def __new__(cls):
         Path(cls.texcache).mkdir(parents=True, exist_ok=True)
         return object.__new__(cls)
 
+    @_api.deprecated("3.6")
     def get_font_config(self):
+        preamble, font_cmd = self._get_font_preamble_and_command()
+        # Add a hash of the latex preamble to fontconfig so that the
+        # correct png is selected for strings rendered with same font and dpi
+        # even if the latex preamble changes within the session
+        preambles = preamble + font_cmd + self.get_custom_preamble()
+        return hashlib.md5(preambles.encode('utf-8')).hexdigest()
+
+    @classmethod
+    def _get_font_family_and_reduced(cls):
```

Comment:
```suggestion
    def _get_font_family_and_reduced(cls):
        """Return the font family name and whether the font is reduced."""
```
makes it a bit clearer what the second parameter is.